### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by harshalini**
Made vivir fashion responsive for mobile view.
Check it here ->
https://festive-northcutt-1e57d0.netlify.app/
PicsArt_02-17-12 12 03
PicsArt_02-17-12 12 42
PicsArt_02-17-12 13 18
PicsArt_02-17-12 13 34
PicsArt_02-17-12 14 01
PicsArt_02-17-12 14 22

### CODE REVIEW

File: products.css [hunk 1]
```diff
@@ -34,6 +38,28 @@
     margin-top: -1.5rem;
     height: 100vh;
     background: radial-gradient(var(--white), var(--light-coral));
+    z-index: 5;
+}
+#toggle-sidebar {
+    visibility: hidden;
+}
+#toggle-sidebar:checked ~ .sidebar {
```
Comment (bhargaviRdy) [inline products.css:46]: using id for css styling is not the best practice

**Review by bhargaviRdy — COMMENTED** [review]
There's horizontal scroll in the mobile view, which creates better experience if not present, other than that everything looks perfect.

File: index.html [hunk 2]
```diff
@@ -16,19 +17,21 @@
...
                 </ul>
             </nav>
         </div>
         <div class="product-container">
             <div class="vivir-header vivir-about">
                 <p class="slogan">Live Up Your Fashion</p>
                 <p class="info">Style yourself with the wide range of exclusive variety! <br> Make your own trends. <br> Express your uniqueness with Vivir!</p>
-                <a href="./products.html"><button class="v-btn" id="explore-btn">Explore now &#8594;</button></a>
+                  <a href="./products.html">
+                    <button class="btn v-btn explore-btn">Explore now</button>
```
Comment (Kothalkarpriya) [inline index.html:33]: needed formatting here

**Review by Kothalkarpriya — APPROVED** [review]
Good work! Given a small suggestion, You can merge

File: products.css (hunk 1 above)
Comment (harshalini) [reply products.css:46]: Actually class was on working on it, that's why used id, please approve this PR

**Review by bhargaviRdy — APPROVED** [review]

**Pull request merged**